### DISCUSSION TITLE
Failed To Ping Server

### DISCUSSION THREAD

**Discussion by bigdoug2005** [advplyr/audiobookshelf#1493] [Q&A] [answered]
I run audiobookshelf on a VPS. I use Nginx Proxy Manager to put audiobookshelf behind a subdomain on my server forwarding to port 13378. I can access ABS using http://audiobook.myDomain.com through a browser. However if I try to use this same address in the iOS app it responds with "Failed to Ping Server". I can successfully connect in the iOS app using the VPS IP address directly on port 13378. 

![image](https://user-images.githubusercontent.com/8703011/217667052-b5d8714e-d06d-405b-952d-42d7bac99434.png)

<img src="https://user-images.githubusercontent.com/8703011/217667200-eb2e095b-adc7-4087-b7ce-18593a3dd3d5.png" width="350" >

**Comment by advplyr** [comment, marked as answer]
Right now the iOS app is restricted to only allowing HTTPS for external connections. This is something from iOS that came as a surprise when updating the deployment target to iOS 14.
We're discussing this in Discord and whether we can solve for it or will require HTTPS going forward.

**Comment by bigdoug2005** [comment]
Thank you for the quick reply! That makes sense, but is unfortunate. I guess I'll look into getting a cert.

Edit: It was as easy as requesting a Let's Encrypt cert on the SSL tab then changing to http**s**://audiobook.mydomain.com
![image](https://user-images.githubusercontent.com/8703011/217669300-8fb231aa-755a-4974-8616-6709dc88fedf.png)

**Reply by advplyr** [reply]
Great! If you are still interested in this issue follow [#539](https://github.com/advplyr/audiobookshelf-app/issues/538)
